klayout/drc/README.md

```diff
@@ -2,44 +2,34 @@
 
 Explains how to use the runset.
 
-## Folder Structure
+## **Folder Structure**
 
 ```text
-📁drc
- ┣ 📁testing
- ┣ 📁rule_decks
- ┣ 📜README.md
- ┗ 📜run_drc.py
+📁 drc
+ ┣ 📁testing                        Testing environment directory for GF180MCU DRC. 
+ ┣ 📁rule_decks                     All DRC rule decks used in GF180MCU.
+ ┣ 📜README.md                      This file to document the DRC run for GF180MCU.
+ ┗ 📜run_drc.py                     Main python script used for GF180MCU DRC.
  ```
 
-## Rule Deck Usage
-The `run_drc.py` script takes a gds file to run DRC rule decks of GF180 technology with switches to select subsets of all checks. 
+## **Prerequisites**
+You need the following set of tools installed to be able to run GF180MCU DRC:
+- Python 3.6+
+- KLayout 0.28.0+
```

Comment:
@FaragElsayed2 Please change the minimum klayout version to 0.28.4

Reply:
Done

Reply:
Updated in run_drc & run_regression python scripts

Reply:
Thanks @FaragElsayed2